### PR TITLE
Add support of heic images

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
Product Note: CommCare will allow `heic` images in the Image Upload questions.

### CODE REVIEW

File: app/src/org/commcare/utils/FormUploadUtil.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
     // This list is also maintained server side and any changes to this should be replicated on server as well
```
Comment (ShivamPokhriyal) [inline app/src/org/commcare/utils/FormUploadUtil.java:57]: I cannot find this list on HQ. Is this line still true?

**Comment by dimagimon** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          6 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not contain any JIRA issue keys in the PR title or commit messages (e.g. KEY-123)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/dimagi/commcare-android/blob/e45b4adc7897f7eb6430620f5d8c965197cc4e6b/app/src/org/commcare/utils/FormUploadUtil.java#L66">app/src/org/commcare/utils/FormUploadUtil.java#L66</a> - <code>Cipher.getInstance</code> should not be called without setting the encryption mode and padding</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/dimagi/commcare-android/blob/e45b4adc7897f7eb6430620f5d8c965197cc4e6b/app/src/org/commcare/utils/FormUploadUtil.java#L138">app/src/org/commcare/utils/FormUploadUtil.java#L138</a> - Using 'Log' instead of 'Timber'</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/dimagi/commcare-android/blob/e45b4adc7897f7eb6430620f5d8c965197cc4e6b/app/src/org/commcare/utils/FormUploadUtil.java#L243">app/src/org/commcare/utils/FormUploadUtil.java#L243</a> - Using 'Log' instead of 'Timber'</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/dimagi/commcare-android/blob/e45b4adc7897f7eb6430620f5d8c965197cc4e6b/app/src/org/commcare/utils/FormUploadUtil.java#L244">app/src/org/commcare/utils/FormUploadUtil.java#L244</a> - Using 'Log' instead of 'Timber'</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/dimagi/commcare-android/blob/e45b4adc7897f7eb6430620f5d8c965197cc4e6b/app/src/org/commcare/utils/FormUploadUtil.java#L301">app/src/org/commcare/utils/FormUploadUtil.java#L301</a> - Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by shubham1g5 — APPROVED** [review]

**Comment by damagatchi** [comment]
[33% (0.0%) vs master 33%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462441/)

**Review by ctsims — APPROVED** [review]

File: app/src/org/commcare/utils/FormUploadUtil.java (hunk 1 above)
Comment (mkangia) [reply app/src/org/commcare/utils/FormUploadUtil.java:57]: we actually didn't add it eventually. It was present on the PR but later reverted. Apologies, i should have notified you.

https://github.com/dimagi/commcare-hq/pull/28458#issuecomment-703786770
So eventually we just added check for the xml file extension.

**Comment by ShivamPokhriyal** [comment]
Closing this until we figure out a better way to show heic images in the imageView

**Pull request closed without merging**